gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1668,7 +1668,7 @@ public static SnapshotListOption fields(SnapshotField... fields) {
   Snapshot getSnapshot(String snapshot, SnapshotOption... options);
 
   /**
-   * Lists all snapshots.
+   * Lists snapshots.
```

Comment:
The reason I didn't do it is that for other resources the `all` is meant to indicate that the list is "aggregated" (on either regions or zones).